src/services/utils.spec.js

```diff
@@ -97,4 +97,11 @@ describe('utils', () => {
       .toEqual(false)
     expect(deepEqual({ r: [circular] }, { r: [circular] })).toEqual(true)
   })
+  it('getProp', () => {
```

Comment:
For `getProp` and `getValueByPath` would be nice to have a brief comment on their purpose.